src/nbody/plotter.py

```diff
@@ -0,0 +1,36 @@
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.animation import FuncAnimation
+
+def plotter(filename):
+
+    # Extracting the positions of each body at each time step
+    df = pd.read_csv(filename)
```

Comment:
we could add something like:
```suggestion
df = pd.read_csv(filename)
required_cols = {"bodies", "time", "px", "py"}
if not required_cols.issubset(df.columns):
       missing = required_cols - set(df.columns)
       raise ValueError(f"CSV missing required columns: {missing}")
```